ecommerce/views.py

```diff
@@ -133,12 +133,8 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         if decision != CYBERSOURCE_DECISION_ACCEPT:
             handle_rejected_order(order=order, decision=decision)
         else:
-            # import pdb; pdb.set_trace()
             complete_successful_order(order)
 
-        # Sync order data with hubspot
-        sync_hubspot_application_from_order(order)
```

Comment:
Hubspot sync should now happen within `handle_rejected_order` or `complete_successful_order`